From: Jens Georg <mail@jensge.org>
Date: Fri, 14 Jan 2022 14:36:40 +0100
Subject: Revert "build: Do not write man page during test"

This reverts commit 8795626a07963a91400773cfe04d4f2f1c37ff14.

This is broken and will make the test fail with weird write fails

xsltproc is weird...

Fixes #68

(cherry picked from commit 8e84846c2df00523b45e192fa72b0bce38661d49)
---
 doc/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/doc/meson.build b/doc/meson.build
index de94aa2..2fd0106 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -66,7 +66,6 @@ if xsltproc.found()
             xsltproc,
             xlstproc_flags,
             '--noout',
-            '--nowrite',
             stylesheet,
             'gupnp-binding-tool.xml'
         ]
